_posts/blog/2018-10-10/2018-10-10-how-to-buy-cryptocurrencies-privately-with-no-fees.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to Buy Cryptocurrencies <span class='post-title-extra'>Privately with no Fees</span>"
+layout: post
+date: '2018-10-10 06:30:00 -0300'
+social_title: 'How to Buy Cryptocurrencies Privately with no Fees'
+social_description: 'This tutorial explains how to use OpenBazaar to purchase cryptocurrencies.'
+---
+
+This tutorial will explain how to purchase cryptocurrencies on OpenBazaar, a decentralized marketplace which never has fees and doesn’t require opening an account. This is for people buying cryptocurrency with a different cryptocurrency, such as Bitcoin. At the moment OpenBazaar doesn’t support people buying cryptocurrency with regular fiat money such as US Dollars or Euros, though it’s a feature that will hopefully be supported in the future.
+
+Here’s a video tutorial explaining the process:
+
+{% include modules/embeded-video.html url="https://www.youtube.com/embed/q6ki3_1Io-U"%}
+
+
+
+## Step #1: Download OpenBazaar
+
+Visit [OpenBazaar.org](https://openbazaar.org/download) and click on the installer for your operating system. Once downloaded, open the file and it will begin the process of installing OpenBazaar.
+<br>  
+## Step #2: Select cryptocurrency
+
+You will be asked which cryptocurrency you want to use. This is the cryptocurrency that you will pay with as a buyer, or receive as a seller. This is not the currency that you wish to purchase.
+<br>  
+![Select Purchasing Cryptocurrency](selectingcurrency.gif "Select Purchasing Cryptocurrency")
+<br>  
+At the moment OpenBazaar users are limited to using only one cryptocurrency at a time, and can only choose BTC, BCH, or ZEC. However in the near future users will be able to use multiple currencies at the same time, and will have LTC, ETH, and other coins available to use.
+<br>  
+## Step #3: Complete Onboarding
+
+Go through the onboarding process and enter whatever information you choose. If you want to remain private then keep the randomly generated name assigned to you. Choose the currency you want to have prices displayed in; you’ll want this to be the currency that you most commonly use so that the prices of goods and services are familiar to you.
+
+Read the terms of service, and if you agree then accept them.
+<br>  
+![Complete Onboarding](onboarding.gif "Complete Onboarding")
+<br>  
+
+## Step #4: Search for cryptocurrency listings
+
+You’re now up and running on OpenBazaar, so let’s find some cryptocurrency listings. Click the “Discover” button on the top right of the client (the blue tent). After selecting a search provider, you’ll now see lots of listings on the network. To filter down to only cryptocurrency offerings, use the search provider’s filters on the left. You’ll now see a list of cryptocurrencies for sale. Find what you want and click on the listing to view it.
```

Comment:
Should we specify to use the product type filter here?

Reply:
This is shown in the gif, and each provider's filters may vary so pointing them to the filters as a whole seems to make sense.